pkg/tests/accounts_test.go

```diff
@@ -452,6 +457,219 @@ func Test_ImplicitAccounts(t *testing.T) {
 	ts.Wait(ts.Nodes()...)
 }
 
+func Test_NegativeBIC(t *testing.T) {
+	ts := testsuite.NewTestSuite(t,
+		testsuite.WithProtocolParametersOptions(
+			iotago.WithTimeProviderOptions(
+				testsuite.GenesisTimeWithOffsetBySlots(200, testsuite.DefaultSlotDurationInSeconds),
+				testsuite.DefaultSlotDurationInSeconds,
+				8,
+			),
+			iotago.WithLivenessOptions(
+				testsuite.DefaultLivenessThresholdLowerBoundInSeconds,
+				testsuite.DefaultLivenessThresholdUpperBoundInSeconds,
+				testsuite.DefaultMinCommittableAge,
+				100,
+				testsuite.DefaultEpochNearingThreshold,
+			),
+		),
+	)
+	defer ts.Shutdown()
+
+	// Add a validator node to the network. This will add a validator account to the snapshot.
+	node1 := ts.AddValidatorNode("node1")
+	node2 := ts.AddNode("node2")
+
+	validatorBIC := iotago.MaxBlockIssuanceCredits / 2
+	// Add a default block issuer to the network. This will add another block issuer account to the snapshot.
+	walletBIC := iotago.BlockIssuanceCredits(499)
+	wallet := ts.AddGenesisWallet("default", node2, walletBIC)
```

Comment:
I think it would be nicer to create two new Wallets in the snapshot, called wallet1 and wallet2 or something, rather than using the default wallet and the wallet attached to node 1.

Reply:
```suggestion
	wallet1 := ts.AddGenesisWallet("wallet 1", node2, walletBIC)
    wallet2 := ts.AddGenesisWallet("wallet 2", node2, walletBIC)
```